ERROR - 2024-09-30 02:10:58 --> Error connecting to the database: mysqli_sql_exception: No connection could be made because the target machine actively refused it in C:\xampp\htdocs\code5\system\Database\MySQLi\Connection.php:186
Stack trace:
#0 C:\xampp\htdocs\code5\system\Database\MySQLi\Connection.php(186): mysqli->real_connect('localhost', 'root', Object(SensitiveParameterValue), 'employee_form', 3306, '', 0)
#1 C:\xampp\htdocs\code5\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\code5\system\Database\BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\code5\system\Database\BaseBuilder.php(1644): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\code5\system\Model.php(286): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\code5\system\BaseModel.php(676): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\xampp\htdocs\code5\app\Controllers\Crud.php(18): CodeIgniter\BaseModel->findAll()
#7 C:\xampp\htdocs\code5\system\CodeIgniter.php(933): App\Controllers\Crud->fetchAll()
#8 C:\xampp\htdocs\code5\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Crud))
#9 C:\xampp\htdocs\code5\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\code5\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#11 C:\xampp\htdocs\code5\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#12 C:\xampp\htdocs\code5\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#13 C:\xampp\htdocs\code5\system\rewrite.php(44): require_once('C:\\xampp\\htdocs...')
#14 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: No connection could be made because the target machine actively refused it in C:\xampp\htdocs\code5\system\Database\MySQLi\Connection.php:231
Stack trace:
#0 C:\xampp\htdocs\code5\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\code5\system\Database\BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\code5\system\Database\BaseBuilder.php(1644): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp\htdocs\code5\system\Model.php(286): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\code5\system\BaseModel.php(676): CodeIgniter\Model->doFindAll(0, 0)
#5 C:\xampp\htdocs\code5\app\Controllers\Crud.php(18): CodeIgniter\BaseModel->findAll()
#6 C:\xampp\htdocs\code5\system\CodeIgniter.php(933): App\Controllers\Crud->fetchAll()
#7 C:\xampp\htdocs\code5\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Crud))
#8 C:\xampp\htdocs\code5\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\code5\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#10 C:\xampp\htdocs\code5\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#11 C:\xampp\htdocs\code5\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#12 C:\xampp\htdocs\code5\system\rewrite.php(44): require_once('C:\\xampp\\htdocs...')
#13 {main}
CRITICAL - 2024-09-30 02:10:58 --> CodeIgniter\Database\Exceptions\DatabaseException: Unable to connect to the database.
Main connection [MySQLi]: No connection could be made because the target machine actively refused it
[Method: GET, Route: Crud/fetchAll]
in SYSTEMPATH\Database\BaseConnection.php on line 467.
 1 SYSTEMPATH\Database\BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1644): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `employees`', [], false)
 3 SYSTEMPATH\Model.php(286): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(676): CodeIgniter\Model->doFindAll(0, 0)
 5 APPPATH\Controllers\Crud.php(18): CodeIgniter\BaseModel->findAll()
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Crud->fetchAll()
 7 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Crud))
 8 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
12 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\code5\\public\\index.php')
ERROR - 2024-09-30 02:21:03 --> mysqli_sql_exception: Duplicate column name 'mob' in C:\xampp\htdocs\code5\system\Database\MySQLi\Connection.php:312
Stack trace:
#0 C:\xampp\htdocs\code5\system\Database\MySQLi\Connection.php(312): mysqli->query('ALTER TABLE `em...', 0)
#1 C:\xampp\htdocs\code5\system\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('ALTER TABLE `em...')
#2 C:\xampp\htdocs\code5\system\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('ALTER TABLE `em...')
#3 C:\xampp\htdocs\code5\system\Database\Forge.php(772): CodeIgniter\Database\BaseConnection->query('ALTER TABLE `em...')
#4 C:\xampp\htdocs\code5\app\Database\Migrations\2024-09-30-022017_AddMobColumnToEmployees.php(11): CodeIgniter\Database\Forge->addColumn('employees', Array)
#5 C:\xampp\htdocs\code5\system\Database\MigrationRunner.php(867): App\Database\Migrations\AddMobColumnToEmployees->up()
#6 C:\xampp\htdocs\code5\system\Database\MigrationRunner.php(183): CodeIgniter\Database\MigrationRunner->migrate('up', Object(stdClass))
#7 C:\xampp\htdocs\code5\system\Commands\Database\Migrate.php(85): CodeIgniter\Database\MigrationRunner->latest(NULL)
#8 C:\xampp\htdocs\code5\system\CLI\Commands.php(70): CodeIgniter\Commands\Database\Migrate->run(Array)
#9 C:\xampp\htdocs\code5\system\CLI\Console.php(48): CodeIgniter\CLI\Commands->run('migrate', Array)
#10 C:\xampp\htdocs\code5\system\Boot.php(351): CodeIgniter\CLI\Console->run()
#11 C:\xampp\htdocs\code5\system\Boot.php(104): CodeIgniter\Boot::runCommand(Object(CodeIgniter\CLI\Console))
#12 C:\xampp\htdocs\code5\spark(84): CodeIgniter\Boot::bootSpark(Object(Config\Paths))
#13 {main}
ERROR - 2024-09-30 02:29:09 --> mysqli_sql_exception: Duplicate column name 'mob' in C:\xampp\htdocs\code5\system\Database\MySQLi\Connection.php:312
Stack trace:
#0 C:\xampp\htdocs\code5\system\Database\MySQLi\Connection.php(312): mysqli->query('ALTER TABLE `em...', 0)
#1 C:\xampp\htdocs\code5\system\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('ALTER TABLE `em...')
#2 C:\xampp\htdocs\code5\system\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('ALTER TABLE `em...')
#3 C:\xampp\htdocs\code5\system\Database\Forge.php(772): CodeIgniter\Database\BaseConnection->query('ALTER TABLE `em...')
#4 C:\xampp\htdocs\code5\app\Database\Migrations\2024-09-30-022017_AddMobColumnToEmployees.php(11): CodeIgniter\Database\Forge->addColumn('employees', Array)
#5 C:\xampp\htdocs\code5\system\Database\MigrationRunner.php(867): App\Database\Migrations\AddMobColumnToEmployees->up()
#6 C:\xampp\htdocs\code5\system\Database\MigrationRunner.php(183): CodeIgniter\Database\MigrationRunner->migrate('up', Object(stdClass))
#7 C:\xampp\htdocs\code5\system\Commands\Database\Migrate.php(85): CodeIgniter\Database\MigrationRunner->latest(NULL)
#8 C:\xampp\htdocs\code5\system\CLI\Commands.php(70): CodeIgniter\Commands\Database\Migrate->run(Array)
#9 C:\xampp\htdocs\code5\system\CLI\Console.php(48): CodeIgniter\CLI\Commands->run('migrate', Array)
#10 C:\xampp\htdocs\code5\system\Boot.php(351): CodeIgniter\CLI\Console->run()
#11 C:\xampp\htdocs\code5\system\Boot.php(104): CodeIgniter\Boot::runCommand(Object(CodeIgniter\CLI\Console))
#12 C:\xampp\htdocs\code5\spark(84): CodeIgniter\Boot::bootSpark(Object(Config\Paths))
#13 {main}
ERROR - 2024-09-30 02:30:45 --> mysqli_sql_exception: Duplicate column name 'mob' in C:\xampp\htdocs\code5\system\Database\MySQLi\Connection.php:312
Stack trace:
#0 C:\xampp\htdocs\code5\system\Database\MySQLi\Connection.php(312): mysqli->query('ALTER TABLE `em...', 0)
#1 C:\xampp\htdocs\code5\system\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('ALTER TABLE `em...')
#2 C:\xampp\htdocs\code5\system\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('ALTER TABLE `em...')
#3 C:\xampp\htdocs\code5\system\Database\Forge.php(772): CodeIgniter\Database\BaseConnection->query('ALTER TABLE `em...')
#4 C:\xampp\htdocs\code5\app\Database\Migrations\2024-09-30-022017_AddMobColumnToEmployees.php(11): CodeIgniter\Database\Forge->addColumn('employees', Array)
#5 C:\xampp\htdocs\code5\system\Database\MigrationRunner.php(867): App\Database\Migrations\AddMobColumnToEmployees->up()
#6 C:\xampp\htdocs\code5\system\Database\MigrationRunner.php(183): CodeIgniter\Database\MigrationRunner->migrate('up', Object(stdClass))
#7 C:\xampp\htdocs\code5\system\Commands\Database\Migrate.php(85): CodeIgniter\Database\MigrationRunner->latest(NULL)
#8 C:\xampp\htdocs\code5\system\CLI\Commands.php(70): CodeIgniter\Commands\Database\Migrate->run(Array)
#9 C:\xampp\htdocs\code5\system\CLI\Console.php(48): CodeIgniter\CLI\Commands->run('migrate', Array)
#10 C:\xampp\htdocs\code5\system\Boot.php(351): CodeIgniter\CLI\Console->run()
#11 C:\xampp\htdocs\code5\system\Boot.php(104): CodeIgniter\Boot::runCommand(Object(CodeIgniter\CLI\Console))
#12 C:\xampp\htdocs\code5\spark(84): CodeIgniter\Boot::bootSpark(Object(Config\Paths))
#13 {main}
ERROR - 2024-09-30 02:32:43 --> mysqli_sql_exception: Unknown column 'address' in 'field list' in C:\xampp\htdocs\code5\system\Database\MySQLi\Connection.php:312
Stack trace:
#0 C:\xampp\htdocs\code5\system\Database\MySQLi\Connection.php(312): mysqli->query('INSERT INTO `em...', 0)
#1 C:\xampp\htdocs\code5\system\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `em...')
#2 C:\xampp\htdocs\code5\system\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `em...')
#3 C:\xampp\htdocs\code5\system\Database\BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO `em...', Array, false)
#4 C:\xampp\htdocs\code5\system\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\code5\system\BaseModel.php(840): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\code5\system\Model.php(802): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xampp\htdocs\code5\app\Controllers\Crud.php(32): CodeIgniter\Model->insert(Array)
#8 C:\xampp\htdocs\code5\system\CodeIgniter.php(933): App\Controllers\Crud->create()
#9 C:\xampp\htdocs\code5\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Crud))
#10 C:\xampp\htdocs\code5\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\code5\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#12 C:\xampp\htdocs\code5\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#13 C:\xampp\htdocs\code5\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#14 C:\xampp\htdocs\code5\system\rewrite.php(44): require_once('C:\\xampp\\htdocs...')
#15 {main}
CRITICAL - 2024-09-30 02:32:43 --> CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'address' in 'field list'
[Method: POST, Route: Crud/create]
in SYSTEMPATH\Database\BaseConnection.php on line 688.
 1 SYSTEMPATH\Database\BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO `employees` (`name`, `email`, `mob`, `address`) VALUES (:name:, :email:, :mob:, :address:)', [...], false)
 2 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(840): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(802): CodeIgniter\BaseModel->insert([...], true)
 5 APPPATH\Controllers\Crud.php(32): CodeIgniter\Model->insert([...])
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Crud->create()
 7 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Crud))
 8 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
12 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\code5\\public\\index.php')
CRITICAL - 2024-09-30 02:32:43 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'address' in 'field list'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 317.
 1 SYSTEMPATH\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `employees` (`name`, `email`, `mob`, `address`) VALUES (\'Rishiraj Korde\', \'rishirajkorde@gmail.com\', \'08888172881\', NULL)')
 2 SYSTEMPATH\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `employees` (`name`, `email`, `mob`, `address`) VALUES (\'Rishiraj Korde\', \'rishirajkorde@gmail.com\', \'08888172881\', NULL)')
 3 SYSTEMPATH\Database\BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO `employees` (`name`, `email`, `mob`, `address`) VALUES (:name:, :email:, :mob:, :address:)', [...], false)
 4 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 5 SYSTEMPATH\BaseModel.php(840): CodeIgniter\Model->doInsert([...])
 6 SYSTEMPATH\Model.php(802): CodeIgniter\BaseModel->insert([...], true)
 7 APPPATH\Controllers\Crud.php(32): CodeIgniter\Model->insert([...])
 8 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Crud->create()
 9 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Crud))
10 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
13 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
14 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\code5\\public\\index.php')
CRITICAL - 2024-09-30 02:32:43 --> [Caused by] mysqli_sql_exception: Unknown column 'address' in 'field list'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 312.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(312): mysqli->query('INSERT INTO `employees` (`name`, `email`, `mob`, `address`) VALUES (\'Rishiraj Korde\', \'rishirajkorde@gmail.com\', \'08888172881\', NULL)', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `employees` (`name`, `email`, `mob`, `address`) VALUES (\'Rishiraj Korde\', \'rishirajkorde@gmail.com\', \'08888172881\', NULL)')
 3 SYSTEMPATH\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `employees` (`name`, `email`, `mob`, `address`) VALUES (\'Rishiraj Korde\', \'rishirajkorde@gmail.com\', \'08888172881\', NULL)')
 4 SYSTEMPATH\Database\BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO `employees` (`name`, `email`, `mob`, `address`) VALUES (:name:, :email:, :mob:, :address:)', [...], false)
 5 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH\BaseModel.php(840): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH\Model.php(802): CodeIgniter\BaseModel->insert([...], true)
 8 APPPATH\Controllers\Crud.php(32): CodeIgniter\Model->insert([...])
 9 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Crud->create()
10 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Crud))
11 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
14 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
15 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\code5\\public\\index.php')
ERROR - 2024-09-30 02:32:45 --> mysqli_sql_exception: Unknown column 'address' in 'field list' in C:\xampp\htdocs\code5\system\Database\MySQLi\Connection.php:312
Stack trace:
#0 C:\xampp\htdocs\code5\system\Database\MySQLi\Connection.php(312): mysqli->query('INSERT INTO `em...', 0)
#1 C:\xampp\htdocs\code5\system\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `em...')
#2 C:\xampp\htdocs\code5\system\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `em...')
#3 C:\xampp\htdocs\code5\system\Database\BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO `em...', Array, false)
#4 C:\xampp\htdocs\code5\system\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\code5\system\BaseModel.php(840): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\code5\system\Model.php(802): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xampp\htdocs\code5\app\Controllers\Crud.php(32): CodeIgniter\Model->insert(Array)
#8 C:\xampp\htdocs\code5\system\CodeIgniter.php(933): App\Controllers\Crud->create()
#9 C:\xampp\htdocs\code5\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Crud))
#10 C:\xampp\htdocs\code5\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\code5\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#12 C:\xampp\htdocs\code5\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#13 C:\xampp\htdocs\code5\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#14 C:\xampp\htdocs\code5\system\rewrite.php(44): require_once('C:\\xampp\\htdocs...')
#15 {main}
CRITICAL - 2024-09-30 02:32:45 --> CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'address' in 'field list'
[Method: POST, Route: Crud/create]
in SYSTEMPATH\Database\BaseConnection.php on line 688.
 1 SYSTEMPATH\Database\BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO `employees` (`name`, `email`, `mob`, `address`) VALUES (:name:, :email:, :mob:, :address:)', [...], false)
 2 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(840): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(802): CodeIgniter\BaseModel->insert([...], true)
 5 APPPATH\Controllers\Crud.php(32): CodeIgniter\Model->insert([...])
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Crud->create()
 7 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Crud))
 8 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
12 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\code5\\public\\index.php')
CRITICAL - 2024-09-30 02:32:45 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'address' in 'field list'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 317.
 1 SYSTEMPATH\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `employees` (`name`, `email`, `mob`, `address`) VALUES (\'Rishiraj Korde\', \'rishirajkorde@gmail.com\', \'08888172881\', NULL)')
 2 SYSTEMPATH\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `employees` (`name`, `email`, `mob`, `address`) VALUES (\'Rishiraj Korde\', \'rishirajkorde@gmail.com\', \'08888172881\', NULL)')
 3 SYSTEMPATH\Database\BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO `employees` (`name`, `email`, `mob`, `address`) VALUES (:name:, :email:, :mob:, :address:)', [...], false)
 4 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 5 SYSTEMPATH\BaseModel.php(840): CodeIgniter\Model->doInsert([...])
 6 SYSTEMPATH\Model.php(802): CodeIgniter\BaseModel->insert([...], true)
 7 APPPATH\Controllers\Crud.php(32): CodeIgniter\Model->insert([...])
 8 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Crud->create()
 9 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Crud))
10 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
13 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
14 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\code5\\public\\index.php')
CRITICAL - 2024-09-30 02:32:45 --> [Caused by] mysqli_sql_exception: Unknown column 'address' in 'field list'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 312.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(312): mysqli->query('INSERT INTO `employees` (`name`, `email`, `mob`, `address`) VALUES (\'Rishiraj Korde\', \'rishirajkorde@gmail.com\', \'08888172881\', NULL)', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `employees` (`name`, `email`, `mob`, `address`) VALUES (\'Rishiraj Korde\', \'rishirajkorde@gmail.com\', \'08888172881\', NULL)')
 3 SYSTEMPATH\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `employees` (`name`, `email`, `mob`, `address`) VALUES (\'Rishiraj Korde\', \'rishirajkorde@gmail.com\', \'08888172881\', NULL)')
 4 SYSTEMPATH\Database\BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO `employees` (`name`, `email`, `mob`, `address`) VALUES (:name:, :email:, :mob:, :address:)', [...], false)
 5 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH\BaseModel.php(840): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH\Model.php(802): CodeIgniter\BaseModel->insert([...], true)
 8 APPPATH\Controllers\Crud.php(32): CodeIgniter\Model->insert([...])
 9 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Crud->create()
10 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Crud))
11 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
14 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
15 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\code5\\public\\index.php')
ERROR - 2024-09-30 02:32:46 --> mysqli_sql_exception: Unknown column 'address' in 'field list' in C:\xampp\htdocs\code5\system\Database\MySQLi\Connection.php:312
Stack trace:
#0 C:\xampp\htdocs\code5\system\Database\MySQLi\Connection.php(312): mysqli->query('INSERT INTO `em...', 0)
#1 C:\xampp\htdocs\code5\system\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `em...')
#2 C:\xampp\htdocs\code5\system\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `em...')
#3 C:\xampp\htdocs\code5\system\Database\BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO `em...', Array, false)
#4 C:\xampp\htdocs\code5\system\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\code5\system\BaseModel.php(840): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\code5\system\Model.php(802): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xampp\htdocs\code5\app\Controllers\Crud.php(32): CodeIgniter\Model->insert(Array)
#8 C:\xampp\htdocs\code5\system\CodeIgniter.php(933): App\Controllers\Crud->create()
#9 C:\xampp\htdocs\code5\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Crud))
#10 C:\xampp\htdocs\code5\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\code5\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#12 C:\xampp\htdocs\code5\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#13 C:\xampp\htdocs\code5\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#14 C:\xampp\htdocs\code5\system\rewrite.php(44): require_once('C:\\xampp\\htdocs...')
#15 {main}
CRITICAL - 2024-09-30 02:32:46 --> CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'address' in 'field list'
[Method: POST, Route: Crud/create]
in SYSTEMPATH\Database\BaseConnection.php on line 688.
 1 SYSTEMPATH\Database\BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO `employees` (`name`, `email`, `mob`, `address`) VALUES (:name:, :email:, :mob:, :address:)', [...], false)
 2 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(840): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(802): CodeIgniter\BaseModel->insert([...], true)
 5 APPPATH\Controllers\Crud.php(32): CodeIgniter\Model->insert([...])
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Crud->create()
 7 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Crud))
 8 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
12 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\code5\\public\\index.php')
CRITICAL - 2024-09-30 02:32:46 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'address' in 'field list'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 317.
 1 SYSTEMPATH\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `employees` (`name`, `email`, `mob`, `address`) VALUES (\'Rishiraj Korde\', \'rishirajkorde@gmail.com\', \'08888172881\', NULL)')
 2 SYSTEMPATH\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `employees` (`name`, `email`, `mob`, `address`) VALUES (\'Rishiraj Korde\', \'rishirajkorde@gmail.com\', \'08888172881\', NULL)')
 3 SYSTEMPATH\Database\BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO `employees` (`name`, `email`, `mob`, `address`) VALUES (:name:, :email:, :mob:, :address:)', [...], false)
 4 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 5 SYSTEMPATH\BaseModel.php(840): CodeIgniter\Model->doInsert([...])
 6 SYSTEMPATH\Model.php(802): CodeIgniter\BaseModel->insert([...], true)
 7 APPPATH\Controllers\Crud.php(32): CodeIgniter\Model->insert([...])
 8 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Crud->create()
 9 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Crud))
10 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
13 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
14 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\code5\\public\\index.php')
CRITICAL - 2024-09-30 02:32:46 --> [Caused by] mysqli_sql_exception: Unknown column 'address' in 'field list'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 312.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(312): mysqli->query('INSERT INTO `employees` (`name`, `email`, `mob`, `address`) VALUES (\'Rishiraj Korde\', \'rishirajkorde@gmail.com\', \'08888172881\', NULL)', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `employees` (`name`, `email`, `mob`, `address`) VALUES (\'Rishiraj Korde\', \'rishirajkorde@gmail.com\', \'08888172881\', NULL)')
 3 SYSTEMPATH\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `employees` (`name`, `email`, `mob`, `address`) VALUES (\'Rishiraj Korde\', \'rishirajkorde@gmail.com\', \'08888172881\', NULL)')
 4 SYSTEMPATH\Database\BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO `employees` (`name`, `email`, `mob`, `address`) VALUES (:name:, :email:, :mob:, :address:)', [...], false)
 5 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH\BaseModel.php(840): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH\Model.php(802): CodeIgniter\BaseModel->insert([...], true)
 8 APPPATH\Controllers\Crud.php(32): CodeIgniter\Model->insert([...])
 9 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Crud->create()
10 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Crud))
11 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
14 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
15 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\code5\\public\\index.php')
ERROR - 2024-09-30 02:32:46 --> mysqli_sql_exception: Unknown column 'address' in 'field list' in C:\xampp\htdocs\code5\system\Database\MySQLi\Connection.php:312
Stack trace:
#0 C:\xampp\htdocs\code5\system\Database\MySQLi\Connection.php(312): mysqli->query('INSERT INTO `em...', 0)
#1 C:\xampp\htdocs\code5\system\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `em...')
#2 C:\xampp\htdocs\code5\system\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `em...')
#3 C:\xampp\htdocs\code5\system\Database\BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO `em...', Array, false)
#4 C:\xampp\htdocs\code5\system\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\code5\system\BaseModel.php(840): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\code5\system\Model.php(802): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xampp\htdocs\code5\app\Controllers\Crud.php(32): CodeIgniter\Model->insert(Array)
#8 C:\xampp\htdocs\code5\system\CodeIgniter.php(933): App\Controllers\Crud->create()
#9 C:\xampp\htdocs\code5\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Crud))
#10 C:\xampp\htdocs\code5\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\code5\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#12 C:\xampp\htdocs\code5\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#13 C:\xampp\htdocs\code5\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#14 C:\xampp\htdocs\code5\system\rewrite.php(44): require_once('C:\\xampp\\htdocs...')
#15 {main}
CRITICAL - 2024-09-30 02:32:46 --> CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'address' in 'field list'
[Method: POST, Route: Crud/create]
in SYSTEMPATH\Database\BaseConnection.php on line 688.
 1 SYSTEMPATH\Database\BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO `employees` (`name`, `email`, `mob`, `address`) VALUES (:name:, :email:, :mob:, :address:)', [...], false)
 2 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(840): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(802): CodeIgniter\BaseModel->insert([...], true)
 5 APPPATH\Controllers\Crud.php(32): CodeIgniter\Model->insert([...])
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Crud->create()
 7 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Crud))
 8 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
12 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\code5\\public\\index.php')
CRITICAL - 2024-09-30 02:32:46 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'address' in 'field list'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 317.
 1 SYSTEMPATH\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `employees` (`name`, `email`, `mob`, `address`) VALUES (\'Rishiraj Korde\', \'rishirajkorde@gmail.com\', \'08888172881\', NULL)')
 2 SYSTEMPATH\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `employees` (`name`, `email`, `mob`, `address`) VALUES (\'Rishiraj Korde\', \'rishirajkorde@gmail.com\', \'08888172881\', NULL)')
 3 SYSTEMPATH\Database\BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO `employees` (`name`, `email`, `mob`, `address`) VALUES (:name:, :email:, :mob:, :address:)', [...], false)
 4 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 5 SYSTEMPATH\BaseModel.php(840): CodeIgniter\Model->doInsert([...])
 6 SYSTEMPATH\Model.php(802): CodeIgniter\BaseModel->insert([...], true)
 7 APPPATH\Controllers\Crud.php(32): CodeIgniter\Model->insert([...])
 8 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Crud->create()
 9 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Crud))
10 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
13 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
14 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\code5\\public\\index.php')
CRITICAL - 2024-09-30 02:32:46 --> [Caused by] mysqli_sql_exception: Unknown column 'address' in 'field list'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 312.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(312): mysqli->query('INSERT INTO `employees` (`name`, `email`, `mob`, `address`) VALUES (\'Rishiraj Korde\', \'rishirajkorde@gmail.com\', \'08888172881\', NULL)', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `employees` (`name`, `email`, `mob`, `address`) VALUES (\'Rishiraj Korde\', \'rishirajkorde@gmail.com\', \'08888172881\', NULL)')
 3 SYSTEMPATH\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `employees` (`name`, `email`, `mob`, `address`) VALUES (\'Rishiraj Korde\', \'rishirajkorde@gmail.com\', \'08888172881\', NULL)')
 4 SYSTEMPATH\Database\BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO `employees` (`name`, `email`, `mob`, `address`) VALUES (:name:, :email:, :mob:, :address:)', [...], false)
 5 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH\BaseModel.php(840): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH\Model.php(802): CodeIgniter\BaseModel->insert([...], true)
 8 APPPATH\Controllers\Crud.php(32): CodeIgniter\Model->insert([...])
 9 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Crud->create()
10 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Crud))
11 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
14 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
15 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\code5\\public\\index.php')
ERROR - 2024-09-30 02:32:47 --> mysqli_sql_exception: Unknown column 'address' in 'field list' in C:\xampp\htdocs\code5\system\Database\MySQLi\Connection.php:312
Stack trace:
#0 C:\xampp\htdocs\code5\system\Database\MySQLi\Connection.php(312): mysqli->query('INSERT INTO `em...', 0)
#1 C:\xampp\htdocs\code5\system\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `em...')
#2 C:\xampp\htdocs\code5\system\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `em...')
#3 C:\xampp\htdocs\code5\system\Database\BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO `em...', Array, false)
#4 C:\xampp\htdocs\code5\system\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\code5\system\BaseModel.php(840): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\code5\system\Model.php(802): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xampp\htdocs\code5\app\Controllers\Crud.php(32): CodeIgniter\Model->insert(Array)
#8 C:\xampp\htdocs\code5\system\CodeIgniter.php(933): App\Controllers\Crud->create()
#9 C:\xampp\htdocs\code5\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Crud))
#10 C:\xampp\htdocs\code5\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\code5\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#12 C:\xampp\htdocs\code5\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#13 C:\xampp\htdocs\code5\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#14 C:\xampp\htdocs\code5\system\rewrite.php(44): require_once('C:\\xampp\\htdocs...')
#15 {main}
CRITICAL - 2024-09-30 02:32:47 --> CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'address' in 'field list'
[Method: POST, Route: Crud/create]
in SYSTEMPATH\Database\BaseConnection.php on line 688.
 1 SYSTEMPATH\Database\BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO `employees` (`name`, `email`, `mob`, `address`) VALUES (:name:, :email:, :mob:, :address:)', [...], false)
 2 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(840): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(802): CodeIgniter\BaseModel->insert([...], true)
 5 APPPATH\Controllers\Crud.php(32): CodeIgniter\Model->insert([...])
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Crud->create()
 7 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Crud))
 8 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
12 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\code5\\public\\index.php')
CRITICAL - 2024-09-30 02:32:47 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'address' in 'field list'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 317.
 1 SYSTEMPATH\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `employees` (`name`, `email`, `mob`, `address`) VALUES (\'Rishiraj Korde\', \'rishirajkorde@gmail.com\', \'08888172881\', NULL)')
 2 SYSTEMPATH\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `employees` (`name`, `email`, `mob`, `address`) VALUES (\'Rishiraj Korde\', \'rishirajkorde@gmail.com\', \'08888172881\', NULL)')
 3 SYSTEMPATH\Database\BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO `employees` (`name`, `email`, `mob`, `address`) VALUES (:name:, :email:, :mob:, :address:)', [...], false)
 4 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 5 SYSTEMPATH\BaseModel.php(840): CodeIgniter\Model->doInsert([...])
 6 SYSTEMPATH\Model.php(802): CodeIgniter\BaseModel->insert([...], true)
 7 APPPATH\Controllers\Crud.php(32): CodeIgniter\Model->insert([...])
 8 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Crud->create()
 9 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Crud))
10 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
13 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
14 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\code5\\public\\index.php')
CRITICAL - 2024-09-30 02:32:47 --> [Caused by] mysqli_sql_exception: Unknown column 'address' in 'field list'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 312.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(312): mysqli->query('INSERT INTO `employees` (`name`, `email`, `mob`, `address`) VALUES (\'Rishiraj Korde\', \'rishirajkorde@gmail.com\', \'08888172881\', NULL)', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `employees` (`name`, `email`, `mob`, `address`) VALUES (\'Rishiraj Korde\', \'rishirajkorde@gmail.com\', \'08888172881\', NULL)')
 3 SYSTEMPATH\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `employees` (`name`, `email`, `mob`, `address`) VALUES (\'Rishiraj Korde\', \'rishirajkorde@gmail.com\', \'08888172881\', NULL)')
 4 SYSTEMPATH\Database\BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO `employees` (`name`, `email`, `mob`, `address`) VALUES (:name:, :email:, :mob:, :address:)', [...], false)
 5 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH\BaseModel.php(840): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH\Model.php(802): CodeIgniter\BaseModel->insert([...], true)
 8 APPPATH\Controllers\Crud.php(32): CodeIgniter\Model->insert([...])
 9 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Crud->create()
10 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Crud))
11 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
14 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
15 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\code5\\public\\index.php')
ERROR - 2024-09-30 02:32:47 --> mysqli_sql_exception: Unknown column 'address' in 'field list' in C:\xampp\htdocs\code5\system\Database\MySQLi\Connection.php:312
Stack trace:
#0 C:\xampp\htdocs\code5\system\Database\MySQLi\Connection.php(312): mysqli->query('INSERT INTO `em...', 0)
#1 C:\xampp\htdocs\code5\system\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `em...')
#2 C:\xampp\htdocs\code5\system\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `em...')
#3 C:\xampp\htdocs\code5\system\Database\BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO `em...', Array, false)
#4 C:\xampp\htdocs\code5\system\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\code5\system\BaseModel.php(840): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\code5\system\Model.php(802): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xampp\htdocs\code5\app\Controllers\Crud.php(32): CodeIgniter\Model->insert(Array)
#8 C:\xampp\htdocs\code5\system\CodeIgniter.php(933): App\Controllers\Crud->create()
#9 C:\xampp\htdocs\code5\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Crud))
#10 C:\xampp\htdocs\code5\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\code5\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#12 C:\xampp\htdocs\code5\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#13 C:\xampp\htdocs\code5\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#14 C:\xampp\htdocs\code5\system\rewrite.php(44): require_once('C:\\xampp\\htdocs...')
#15 {main}
CRITICAL - 2024-09-30 02:32:47 --> CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'address' in 'field list'
[Method: POST, Route: Crud/create]
in SYSTEMPATH\Database\BaseConnection.php on line 688.
 1 SYSTEMPATH\Database\BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO `employees` (`name`, `email`, `mob`, `address`) VALUES (:name:, :email:, :mob:, :address:)', [...], false)
 2 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(840): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(802): CodeIgniter\BaseModel->insert([...], true)
 5 APPPATH\Controllers\Crud.php(32): CodeIgniter\Model->insert([...])
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Crud->create()
 7 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Crud))
 8 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
12 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\code5\\public\\index.php')
CRITICAL - 2024-09-30 02:32:47 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'address' in 'field list'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 317.
 1 SYSTEMPATH\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `employees` (`name`, `email`, `mob`, `address`) VALUES (\'Rishiraj Korde\', \'rishirajkorde@gmail.com\', \'08888172881\', NULL)')
 2 SYSTEMPATH\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `employees` (`name`, `email`, `mob`, `address`) VALUES (\'Rishiraj Korde\', \'rishirajkorde@gmail.com\', \'08888172881\', NULL)')
 3 SYSTEMPATH\Database\BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO `employees` (`name`, `email`, `mob`, `address`) VALUES (:name:, :email:, :mob:, :address:)', [...], false)
 4 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 5 SYSTEMPATH\BaseModel.php(840): CodeIgniter\Model->doInsert([...])
 6 SYSTEMPATH\Model.php(802): CodeIgniter\BaseModel->insert([...], true)
 7 APPPATH\Controllers\Crud.php(32): CodeIgniter\Model->insert([...])
 8 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Crud->create()
 9 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Crud))
10 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
13 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
14 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\code5\\public\\index.php')
CRITICAL - 2024-09-30 02:32:47 --> [Caused by] mysqli_sql_exception: Unknown column 'address' in 'field list'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 312.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(312): mysqli->query('INSERT INTO `employees` (`name`, `email`, `mob`, `address`) VALUES (\'Rishiraj Korde\', \'rishirajkorde@gmail.com\', \'08888172881\', NULL)', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `employees` (`name`, `email`, `mob`, `address`) VALUES (\'Rishiraj Korde\', \'rishirajkorde@gmail.com\', \'08888172881\', NULL)')
 3 SYSTEMPATH\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `employees` (`name`, `email`, `mob`, `address`) VALUES (\'Rishiraj Korde\', \'rishirajkorde@gmail.com\', \'08888172881\', NULL)')
 4 SYSTEMPATH\Database\BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO `employees` (`name`, `email`, `mob`, `address`) VALUES (:name:, :email:, :mob:, :address:)', [...], false)
 5 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH\BaseModel.php(840): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH\Model.php(802): CodeIgniter\BaseModel->insert([...], true)
 8 APPPATH\Controllers\Crud.php(32): CodeIgniter\Model->insert([...])
 9 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Crud->create()
10 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Crud))
11 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
14 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
15 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\code5\\public\\index.php')
ERROR - 2024-09-30 02:32:48 --> mysqli_sql_exception: Unknown column 'address' in 'field list' in C:\xampp\htdocs\code5\system\Database\MySQLi\Connection.php:312
Stack trace:
#0 C:\xampp\htdocs\code5\system\Database\MySQLi\Connection.php(312): mysqli->query('INSERT INTO `em...', 0)
#1 C:\xampp\htdocs\code5\system\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `em...')
#2 C:\xampp\htdocs\code5\system\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `em...')
#3 C:\xampp\htdocs\code5\system\Database\BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO `em...', Array, false)
#4 C:\xampp\htdocs\code5\system\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\code5\system\BaseModel.php(840): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\code5\system\Model.php(802): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xampp\htdocs\code5\app\Controllers\Crud.php(32): CodeIgniter\Model->insert(Array)
#8 C:\xampp\htdocs\code5\system\CodeIgniter.php(933): App\Controllers\Crud->create()
#9 C:\xampp\htdocs\code5\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Crud))
#10 C:\xampp\htdocs\code5\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\code5\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#12 C:\xampp\htdocs\code5\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#13 C:\xampp\htdocs\code5\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#14 C:\xampp\htdocs\code5\system\rewrite.php(44): require_once('C:\\xampp\\htdocs...')
#15 {main}
CRITICAL - 2024-09-30 02:32:48 --> CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'address' in 'field list'
[Method: POST, Route: Crud/create]
in SYSTEMPATH\Database\BaseConnection.php on line 688.
 1 SYSTEMPATH\Database\BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO `employees` (`name`, `email`, `mob`, `address`) VALUES (:name:, :email:, :mob:, :address:)', [...], false)
 2 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(840): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(802): CodeIgniter\BaseModel->insert([...], true)
 5 APPPATH\Controllers\Crud.php(32): CodeIgniter\Model->insert([...])
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Crud->create()
 7 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Crud))
 8 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
12 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\code5\\public\\index.php')
CRITICAL - 2024-09-30 02:32:48 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'address' in 'field list'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 317.
 1 SYSTEMPATH\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `employees` (`name`, `email`, `mob`, `address`) VALUES (\'Rishiraj Korde\', \'rishirajkorde@gmail.com\', \'08888172881\', NULL)')
 2 SYSTEMPATH\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `employees` (`name`, `email`, `mob`, `address`) VALUES (\'Rishiraj Korde\', \'rishirajkorde@gmail.com\', \'08888172881\', NULL)')
 3 SYSTEMPATH\Database\BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO `employees` (`name`, `email`, `mob`, `address`) VALUES (:name:, :email:, :mob:, :address:)', [...], false)
 4 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 5 SYSTEMPATH\BaseModel.php(840): CodeIgniter\Model->doInsert([...])
 6 SYSTEMPATH\Model.php(802): CodeIgniter\BaseModel->insert([...], true)
 7 APPPATH\Controllers\Crud.php(32): CodeIgniter\Model->insert([...])
 8 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Crud->create()
 9 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Crud))
10 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
13 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
14 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\code5\\public\\index.php')
CRITICAL - 2024-09-30 02:32:48 --> [Caused by] mysqli_sql_exception: Unknown column 'address' in 'field list'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 312.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(312): mysqli->query('INSERT INTO `employees` (`name`, `email`, `mob`, `address`) VALUES (\'Rishiraj Korde\', \'rishirajkorde@gmail.com\', \'08888172881\', NULL)', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `employees` (`name`, `email`, `mob`, `address`) VALUES (\'Rishiraj Korde\', \'rishirajkorde@gmail.com\', \'08888172881\', NULL)')
 3 SYSTEMPATH\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `employees` (`name`, `email`, `mob`, `address`) VALUES (\'Rishiraj Korde\', \'rishirajkorde@gmail.com\', \'08888172881\', NULL)')
 4 SYSTEMPATH\Database\BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO `employees` (`name`, `email`, `mob`, `address`) VALUES (:name:, :email:, :mob:, :address:)', [...], false)
 5 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH\BaseModel.php(840): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH\Model.php(802): CodeIgniter\BaseModel->insert([...], true)
 8 APPPATH\Controllers\Crud.php(32): CodeIgniter\Model->insert([...])
 9 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Crud->create()
10 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Crud))
11 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
14 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
15 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\code5\\public\\index.php')
ERROR - 2024-09-30 02:32:57 --> mysqli_sql_exception: Duplicate column name 'mob' in C:\xampp\htdocs\code5\system\Database\MySQLi\Connection.php:312
Stack trace:
#0 C:\xampp\htdocs\code5\system\Database\MySQLi\Connection.php(312): mysqli->query('ALTER TABLE `em...', 0)
#1 C:\xampp\htdocs\code5\system\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('ALTER TABLE `em...')
#2 C:\xampp\htdocs\code5\system\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('ALTER TABLE `em...')
#3 C:\xampp\htdocs\code5\system\Database\Forge.php(772): CodeIgniter\Database\BaseConnection->query('ALTER TABLE `em...')
#4 C:\xampp\htdocs\code5\app\Database\Migrations\2024-09-30-022017_AddMobColumnToEmployees.php(11): CodeIgniter\Database\Forge->addColumn('employees', Array)
#5 C:\xampp\htdocs\code5\system\Database\MigrationRunner.php(867): App\Database\Migrations\AddMobColumnToEmployees->up()
#6 C:\xampp\htdocs\code5\system\Database\MigrationRunner.php(183): CodeIgniter\Database\MigrationRunner->migrate('up', Object(stdClass))
#7 C:\xampp\htdocs\code5\system\Commands\Database\Migrate.php(85): CodeIgniter\Database\MigrationRunner->latest(NULL)
#8 C:\xampp\htdocs\code5\system\CLI\Commands.php(70): CodeIgniter\Commands\Database\Migrate->run(Array)
#9 C:\xampp\htdocs\code5\system\CLI\Console.php(48): CodeIgniter\CLI\Commands->run('migrate', Array)
#10 C:\xampp\htdocs\code5\system\Boot.php(351): CodeIgniter\CLI\Console->run()
#11 C:\xampp\htdocs\code5\system\Boot.php(104): CodeIgniter\Boot::runCommand(Object(CodeIgniter\CLI\Console))
#12 C:\xampp\htdocs\code5\spark(84): CodeIgniter\Boot::bootSpark(Object(Config\Paths))
#13 {main}
ERROR - 2024-09-30 02:33:29 --> mysqli_sql_exception: Duplicate column name 'mob' in C:\xampp\htdocs\code5\system\Database\MySQLi\Connection.php:312
Stack trace:
#0 C:\xampp\htdocs\code5\system\Database\MySQLi\Connection.php(312): mysqli->query('ALTER TABLE `em...', 0)
#1 C:\xampp\htdocs\code5\system\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('ALTER TABLE `em...')
#2 C:\xampp\htdocs\code5\system\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('ALTER TABLE `em...')
#3 C:\xampp\htdocs\code5\system\Database\Forge.php(772): CodeIgniter\Database\BaseConnection->query('ALTER TABLE `em...')
#4 C:\xampp\htdocs\code5\app\Database\Migrations\2024-09-30-022017_AddMobColumnToEmployees.php(11): CodeIgniter\Database\Forge->addColumn('employees', Array)
#5 C:\xampp\htdocs\code5\system\Database\MigrationRunner.php(867): App\Database\Migrations\AddMobColumnToEmployees->up()
#6 C:\xampp\htdocs\code5\system\Database\MigrationRunner.php(183): CodeIgniter\Database\MigrationRunner->migrate('up', Object(stdClass))
#7 C:\xampp\htdocs\code5\system\Commands\Database\Migrate.php(85): CodeIgniter\Database\MigrationRunner->latest(NULL)
#8 C:\xampp\htdocs\code5\system\CLI\Commands.php(70): CodeIgniter\Commands\Database\Migrate->run(Array)
#9 C:\xampp\htdocs\code5\system\CLI\Console.php(48): CodeIgniter\CLI\Commands->run('migrate', Array)
#10 C:\xampp\htdocs\code5\system\Boot.php(351): CodeIgniter\CLI\Console->run()
#11 C:\xampp\htdocs\code5\system\Boot.php(104): CodeIgniter\Boot::runCommand(Object(CodeIgniter\CLI\Console))
#12 C:\xampp\htdocs\code5\spark(84): CodeIgniter\Boot::bootSpark(Object(Config\Paths))
#13 {main}
ERROR - 2024-09-30 02:33:49 --> mysqli_sql_exception: Unknown column 'address' in 'field list' in C:\xampp\htdocs\code5\system\Database\MySQLi\Connection.php:312
Stack trace:
#0 C:\xampp\htdocs\code5\system\Database\MySQLi\Connection.php(312): mysqli->query('INSERT INTO `em...', 0)
#1 C:\xampp\htdocs\code5\system\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `em...')
#2 C:\xampp\htdocs\code5\system\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `em...')
#3 C:\xampp\htdocs\code5\system\Database\BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO `em...', Array, false)
#4 C:\xampp\htdocs\code5\system\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\code5\system\BaseModel.php(840): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\code5\system\Model.php(802): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xampp\htdocs\code5\app\Controllers\Crud.php(32): CodeIgniter\Model->insert(Array)
#8 C:\xampp\htdocs\code5\system\CodeIgniter.php(933): App\Controllers\Crud->create()
#9 C:\xampp\htdocs\code5\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Crud))
#10 C:\xampp\htdocs\code5\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\code5\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#12 C:\xampp\htdocs\code5\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#13 C:\xampp\htdocs\code5\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#14 C:\xampp\htdocs\code5\system\rewrite.php(44): require_once('C:\\xampp\\htdocs...')
#15 {main}
CRITICAL - 2024-09-30 02:33:49 --> CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'address' in 'field list'
[Method: POST, Route: Crud/create]
in SYSTEMPATH\Database\BaseConnection.php on line 688.
 1 SYSTEMPATH\Database\BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO `employees` (`name`, `email`, `mob`, `address`) VALUES (:name:, :email:, :mob:, :address:)', [...], false)
 2 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(840): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(802): CodeIgniter\BaseModel->insert([...], true)
 5 APPPATH\Controllers\Crud.php(32): CodeIgniter\Model->insert([...])
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Crud->create()
 7 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Crud))
 8 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
12 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\code5\\public\\index.php')
CRITICAL - 2024-09-30 02:33:49 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'address' in 'field list'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 317.
 1 SYSTEMPATH\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `employees` (`name`, `email`, `mob`, `address`) VALUES (\'Rishiraj Korde\', \'rishirajkorde@gmail.com\', \'08888172881\', NULL)')
 2 SYSTEMPATH\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `employees` (`name`, `email`, `mob`, `address`) VALUES (\'Rishiraj Korde\', \'rishirajkorde@gmail.com\', \'08888172881\', NULL)')
 3 SYSTEMPATH\Database\BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO `employees` (`name`, `email`, `mob`, `address`) VALUES (:name:, :email:, :mob:, :address:)', [...], false)
 4 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 5 SYSTEMPATH\BaseModel.php(840): CodeIgniter\Model->doInsert([...])
 6 SYSTEMPATH\Model.php(802): CodeIgniter\BaseModel->insert([...], true)
 7 APPPATH\Controllers\Crud.php(32): CodeIgniter\Model->insert([...])
 8 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Crud->create()
 9 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Crud))
10 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
13 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
14 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\code5\\public\\index.php')
CRITICAL - 2024-09-30 02:33:49 --> [Caused by] mysqli_sql_exception: Unknown column 'address' in 'field list'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 312.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(312): mysqli->query('INSERT INTO `employees` (`name`, `email`, `mob`, `address`) VALUES (\'Rishiraj Korde\', \'rishirajkorde@gmail.com\', \'08888172881\', NULL)', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `employees` (`name`, `email`, `mob`, `address`) VALUES (\'Rishiraj Korde\', \'rishirajkorde@gmail.com\', \'08888172881\', NULL)')
 3 SYSTEMPATH\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `employees` (`name`, `email`, `mob`, `address`) VALUES (\'Rishiraj Korde\', \'rishirajkorde@gmail.com\', \'08888172881\', NULL)')
 4 SYSTEMPATH\Database\BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO `employees` (`name`, `email`, `mob`, `address`) VALUES (:name:, :email:, :mob:, :address:)', [...], false)
 5 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH\BaseModel.php(840): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH\Model.php(802): CodeIgniter\BaseModel->insert([...], true)
 8 APPPATH\Controllers\Crud.php(32): CodeIgniter\Model->insert([...])
 9 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Crud->create()
10 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Crud))
11 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
14 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
15 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\code5\\public\\index.php')
ERROR - 2024-09-30 02:36:09 --> Error connecting to the database: mysqli_sql_exception: Unknown database 'employee_form' in C:\xampp\htdocs\code5\system\Database\MySQLi\Connection.php:186
Stack trace:
#0 C:\xampp\htdocs\code5\system\Database\MySQLi\Connection.php(186): mysqli->real_connect('localhost', 'root', Object(SensitiveParameterValue), 'employee_form', 3306, '', 0)
#1 C:\xampp\htdocs\code5\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\code5\system\Database\BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\code5\system\Database\BaseBuilder.php(1644): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\code5\system\Model.php(286): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\code5\system\BaseModel.php(676): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\xampp\htdocs\code5\app\Controllers\Crud.php(18): CodeIgniter\BaseModel->findAll()
#7 C:\xampp\htdocs\code5\system\CodeIgniter.php(933): App\Controllers\Crud->fetchAll()
#8 C:\xampp\htdocs\code5\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Crud))
#9 C:\xampp\htdocs\code5\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\code5\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#11 C:\xampp\htdocs\code5\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#12 C:\xampp\htdocs\code5\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#13 C:\xampp\htdocs\code5\system\rewrite.php(44): require_once('C:\\xampp\\htdocs...')
#14 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Unknown database 'employee_form' in C:\xampp\htdocs\code5\system\Database\MySQLi\Connection.php:231
Stack trace:
#0 C:\xampp\htdocs\code5\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\code5\system\Database\BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\code5\system\Database\BaseBuilder.php(1644): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp\htdocs\code5\system\Model.php(286): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\code5\system\BaseModel.php(676): CodeIgniter\Model->doFindAll(0, 0)
#5 C:\xampp\htdocs\code5\app\Controllers\Crud.php(18): CodeIgniter\BaseModel->findAll()
#6 C:\xampp\htdocs\code5\system\CodeIgniter.php(933): App\Controllers\Crud->fetchAll()
#7 C:\xampp\htdocs\code5\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Crud))
#8 C:\xampp\htdocs\code5\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\code5\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#10 C:\xampp\htdocs\code5\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#11 C:\xampp\htdocs\code5\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#12 C:\xampp\htdocs\code5\system\rewrite.php(44): require_once('C:\\xampp\\htdocs...')
#13 {main}
CRITICAL - 2024-09-30 02:36:09 --> CodeIgniter\Database\Exceptions\DatabaseException: Unable to connect to the database.
Main connection [MySQLi]: Unknown database 'employee_form'
[Method: GET, Route: Crud/fetchAll]
in SYSTEMPATH\Database\BaseConnection.php on line 467.
 1 SYSTEMPATH\Database\BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1644): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `employees`', [], false)
 3 SYSTEMPATH\Model.php(286): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(676): CodeIgniter\Model->doFindAll(0, 0)
 5 APPPATH\Controllers\Crud.php(18): CodeIgniter\BaseModel->findAll()
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Crud->fetchAll()
 7 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Crud))
 8 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
12 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\code5\\public\\index.php')
ERROR - 2024-09-30 02:36:11 --> Error connecting to the database: mysqli_sql_exception: Unknown database 'employee_form' in C:\xampp\htdocs\code5\system\Database\MySQLi\Connection.php:186
Stack trace:
#0 C:\xampp\htdocs\code5\system\Database\MySQLi\Connection.php(186): mysqli->real_connect('localhost', 'root', Object(SensitiveParameterValue), 'employee_form', 3306, '', 0)
#1 C:\xampp\htdocs\code5\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\code5\system\Database\BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\code5\system\Database\BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO `em...', Array, false)
#4 C:\xampp\htdocs\code5\system\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\code5\system\BaseModel.php(840): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\code5\system\Model.php(802): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xampp\htdocs\code5\app\Controllers\Crud.php(32): CodeIgniter\Model->insert(Array)
#8 C:\xampp\htdocs\code5\system\CodeIgniter.php(933): App\Controllers\Crud->create()
#9 C:\xampp\htdocs\code5\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Crud))
#10 C:\xampp\htdocs\code5\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\code5\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#12 C:\xampp\htdocs\code5\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#13 C:\xampp\htdocs\code5\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#14 C:\xampp\htdocs\code5\system\rewrite.php(44): require_once('C:\\xampp\\htdocs...')
#15 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Unknown database 'employee_form' in C:\xampp\htdocs\code5\system\Database\MySQLi\Connection.php:231
Stack trace:
#0 C:\xampp\htdocs\code5\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\code5\system\Database\BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\code5\system\Database\BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO `em...', Array, false)
#3 C:\xampp\htdocs\code5\system\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
#4 C:\xampp\htdocs\code5\system\BaseModel.php(840): CodeIgniter\Model->doInsert(Array)
#5 C:\xampp\htdocs\code5\system\Model.php(802): CodeIgniter\BaseModel->insert(Array, true)
#6 C:\xampp\htdocs\code5\app\Controllers\Crud.php(32): CodeIgniter\Model->insert(Array)
#7 C:\xampp\htdocs\code5\system\CodeIgniter.php(933): App\Controllers\Crud->create()
#8 C:\xampp\htdocs\code5\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Crud))
#9 C:\xampp\htdocs\code5\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\code5\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#11 C:\xampp\htdocs\code5\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#12 C:\xampp\htdocs\code5\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#13 C:\xampp\htdocs\code5\system\rewrite.php(44): require_once('C:\\xampp\\htdocs...')
#14 {main}
CRITICAL - 2024-09-30 02:36:11 --> CodeIgniter\Database\Exceptions\DatabaseException: Unable to connect to the database.
Main connection [MySQLi]: Unknown database 'employee_form'
[Method: POST, Route: Crud/create]
in SYSTEMPATH\Database\BaseConnection.php on line 467.
 1 SYSTEMPATH\Database\BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO `employees` (`name`, `email`, `mob`, `address`) VALUES (:name:, :email:, :mob:, :address:)', [...], false)
 3 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 4 SYSTEMPATH\BaseModel.php(840): CodeIgniter\Model->doInsert([...])
 5 SYSTEMPATH\Model.php(802): CodeIgniter\BaseModel->insert([...], true)
 6 APPPATH\Controllers\Crud.php(32): CodeIgniter\Model->insert([...])
 7 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Crud->create()
 8 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Crud))
 9 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
12 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
13 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\code5\\public\\index.php')
ERROR - 2024-09-30 02:36:12 --> Error connecting to the database: mysqli_sql_exception: Unknown database 'employee_form' in C:\xampp\htdocs\code5\system\Database\MySQLi\Connection.php:186
Stack trace:
#0 C:\xampp\htdocs\code5\system\Database\MySQLi\Connection.php(186): mysqli->real_connect('localhost', 'root', Object(SensitiveParameterValue), 'employee_form', 3306, '', 0)
#1 C:\xampp\htdocs\code5\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\code5\system\Database\BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\code5\system\Database\BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO `em...', Array, false)
#4 C:\xampp\htdocs\code5\system\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\code5\system\BaseModel.php(840): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\code5\system\Model.php(802): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xampp\htdocs\code5\app\Controllers\Crud.php(32): CodeIgniter\Model->insert(Array)
#8 C:\xampp\htdocs\code5\system\CodeIgniter.php(933): App\Controllers\Crud->create()
#9 C:\xampp\htdocs\code5\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Crud))
#10 C:\xampp\htdocs\code5\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\code5\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#12 C:\xampp\htdocs\code5\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#13 C:\xampp\htdocs\code5\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#14 C:\xampp\htdocs\code5\system\rewrite.php(44): require_once('C:\\xampp\\htdocs...')
#15 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Unknown database 'employee_form' in C:\xampp\htdocs\code5\system\Database\MySQLi\Connection.php:231
Stack trace:
#0 C:\xampp\htdocs\code5\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\code5\system\Database\BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\code5\system\Database\BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO `em...', Array, false)
#3 C:\xampp\htdocs\code5\system\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
#4 C:\xampp\htdocs\code5\system\BaseModel.php(840): CodeIgniter\Model->doInsert(Array)
#5 C:\xampp\htdocs\code5\system\Model.php(802): CodeIgniter\BaseModel->insert(Array, true)
#6 C:\xampp\htdocs\code5\app\Controllers\Crud.php(32): CodeIgniter\Model->insert(Array)
#7 C:\xampp\htdocs\code5\system\CodeIgniter.php(933): App\Controllers\Crud->create()
#8 C:\xampp\htdocs\code5\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Crud))
#9 C:\xampp\htdocs\code5\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\code5\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#11 C:\xampp\htdocs\code5\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#12 C:\xampp\htdocs\code5\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#13 C:\xampp\htdocs\code5\system\rewrite.php(44): require_once('C:\\xampp\\htdocs...')
#14 {main}
CRITICAL - 2024-09-30 02:36:12 --> CodeIgniter\Database\Exceptions\DatabaseException: Unable to connect to the database.
Main connection [MySQLi]: Unknown database 'employee_form'
[Method: POST, Route: Crud/create]
in SYSTEMPATH\Database\BaseConnection.php on line 467.
 1 SYSTEMPATH\Database\BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO `employees` (`name`, `email`, `mob`, `address`) VALUES (:name:, :email:, :mob:, :address:)', [...], false)
 3 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 4 SYSTEMPATH\BaseModel.php(840): CodeIgniter\Model->doInsert([...])
 5 SYSTEMPATH\Model.php(802): CodeIgniter\BaseModel->insert([...], true)
 6 APPPATH\Controllers\Crud.php(32): CodeIgniter\Model->insert([...])
 7 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Crud->create()
 8 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Crud))
 9 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
12 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
13 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\code5\\public\\index.php')
ERROR - 2024-09-30 02:36:12 --> Error connecting to the database: mysqli_sql_exception: Unknown database 'employee_form' in C:\xampp\htdocs\code5\system\Database\MySQLi\Connection.php:186
Stack trace:
#0 C:\xampp\htdocs\code5\system\Database\MySQLi\Connection.php(186): mysqli->real_connect('localhost', 'root', Object(SensitiveParameterValue), 'employee_form', 3306, '', 0)
#1 C:\xampp\htdocs\code5\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\code5\system\Database\BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\code5\system\Database\BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO `em...', Array, false)
#4 C:\xampp\htdocs\code5\system\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\code5\system\BaseModel.php(840): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\code5\system\Model.php(802): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xampp\htdocs\code5\app\Controllers\Crud.php(32): CodeIgniter\Model->insert(Array)
#8 C:\xampp\htdocs\code5\system\CodeIgniter.php(933): App\Controllers\Crud->create()
#9 C:\xampp\htdocs\code5\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Crud))
#10 C:\xampp\htdocs\code5\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\code5\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#12 C:\xampp\htdocs\code5\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#13 C:\xampp\htdocs\code5\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#14 C:\xampp\htdocs\code5\system\rewrite.php(44): require_once('C:\\xampp\\htdocs...')
#15 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Unknown database 'employee_form' in C:\xampp\htdocs\code5\system\Database\MySQLi\Connection.php:231
Stack trace:
#0 C:\xampp\htdocs\code5\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\code5\system\Database\BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\code5\system\Database\BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO `em...', Array, false)
#3 C:\xampp\htdocs\code5\system\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
#4 C:\xampp\htdocs\code5\system\BaseModel.php(840): CodeIgniter\Model->doInsert(Array)
#5 C:\xampp\htdocs\code5\system\Model.php(802): CodeIgniter\BaseModel->insert(Array, true)
#6 C:\xampp\htdocs\code5\app\Controllers\Crud.php(32): CodeIgniter\Model->insert(Array)
#7 C:\xampp\htdocs\code5\system\CodeIgniter.php(933): App\Controllers\Crud->create()
#8 C:\xampp\htdocs\code5\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Crud))
#9 C:\xampp\htdocs\code5\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\code5\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#11 C:\xampp\htdocs\code5\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#12 C:\xampp\htdocs\code5\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#13 C:\xampp\htdocs\code5\system\rewrite.php(44): require_once('C:\\xampp\\htdocs...')
#14 {main}
CRITICAL - 2024-09-30 02:36:12 --> CodeIgniter\Database\Exceptions\DatabaseException: Unable to connect to the database.
Main connection [MySQLi]: Unknown database 'employee_form'
[Method: POST, Route: Crud/create]
in SYSTEMPATH\Database\BaseConnection.php on line 467.
 1 SYSTEMPATH\Database\BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO `employees` (`name`, `email`, `mob`, `address`) VALUES (:name:, :email:, :mob:, :address:)', [...], false)
 3 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 4 SYSTEMPATH\BaseModel.php(840): CodeIgniter\Model->doInsert([...])
 5 SYSTEMPATH\Model.php(802): CodeIgniter\BaseModel->insert([...], true)
 6 APPPATH\Controllers\Crud.php(32): CodeIgniter\Model->insert([...])
 7 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Crud->create()
 8 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Crud))
 9 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
12 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
13 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\code5\\public\\index.php')
ERROR - 2024-09-30 02:36:12 --> Error connecting to the database: mysqli_sql_exception: Unknown database 'employee_form' in C:\xampp\htdocs\code5\system\Database\MySQLi\Connection.php:186
Stack trace:
#0 C:\xampp\htdocs\code5\system\Database\MySQLi\Connection.php(186): mysqli->real_connect('localhost', 'root', Object(SensitiveParameterValue), 'employee_form', 3306, '', 0)
#1 C:\xampp\htdocs\code5\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\code5\system\Database\BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\code5\system\Database\BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO `em...', Array, false)
#4 C:\xampp\htdocs\code5\system\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\code5\system\BaseModel.php(840): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\code5\system\Model.php(802): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xampp\htdocs\code5\app\Controllers\Crud.php(32): CodeIgniter\Model->insert(Array)
#8 C:\xampp\htdocs\code5\system\CodeIgniter.php(933): App\Controllers\Crud->create()
#9 C:\xampp\htdocs\code5\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Crud))
#10 C:\xampp\htdocs\code5\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\code5\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#12 C:\xampp\htdocs\code5\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#13 C:\xampp\htdocs\code5\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#14 C:\xampp\htdocs\code5\system\rewrite.php(44): require_once('C:\\xampp\\htdocs...')
#15 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Unknown database 'employee_form' in C:\xampp\htdocs\code5\system\Database\MySQLi\Connection.php:231
Stack trace:
#0 C:\xampp\htdocs\code5\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\code5\system\Database\BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\code5\system\Database\BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO `em...', Array, false)
#3 C:\xampp\htdocs\code5\system\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
#4 C:\xampp\htdocs\code5\system\BaseModel.php(840): CodeIgniter\Model->doInsert(Array)
#5 C:\xampp\htdocs\code5\system\Model.php(802): CodeIgniter\BaseModel->insert(Array, true)
#6 C:\xampp\htdocs\code5\app\Controllers\Crud.php(32): CodeIgniter\Model->insert(Array)
#7 C:\xampp\htdocs\code5\system\CodeIgniter.php(933): App\Controllers\Crud->create()
#8 C:\xampp\htdocs\code5\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Crud))
#9 C:\xampp\htdocs\code5\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\code5\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#11 C:\xampp\htdocs\code5\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#12 C:\xampp\htdocs\code5\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#13 C:\xampp\htdocs\code5\system\rewrite.php(44): require_once('C:\\xampp\\htdocs...')
#14 {main}
CRITICAL - 2024-09-30 02:36:12 --> CodeIgniter\Database\Exceptions\DatabaseException: Unable to connect to the database.
Main connection [MySQLi]: Unknown database 'employee_form'
[Method: POST, Route: Crud/create]
in SYSTEMPATH\Database\BaseConnection.php on line 467.
 1 SYSTEMPATH\Database\BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO `employees` (`name`, `email`, `mob`, `address`) VALUES (:name:, :email:, :mob:, :address:)', [...], false)
 3 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 4 SYSTEMPATH\BaseModel.php(840): CodeIgniter\Model->doInsert([...])
 5 SYSTEMPATH\Model.php(802): CodeIgniter\BaseModel->insert([...], true)
 6 APPPATH\Controllers\Crud.php(32): CodeIgniter\Model->insert([...])
 7 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Crud->create()
 8 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Crud))
 9 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
12 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
13 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\code5\\public\\index.php')
ERROR - 2024-09-30 02:37:09 --> Error connecting to the database: mysqli_sql_exception: Unknown database 'employee_form' in C:\xampp\htdocs\code5\system\Database\MySQLi\Connection.php:186
Stack trace:
#0 C:\xampp\htdocs\code5\system\Database\MySQLi\Connection.php(186): mysqli->real_connect('localhost', 'root', Object(SensitiveParameterValue), 'employee_form', 3306, '', 0)
#1 C:\xampp\htdocs\code5\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\code5\system\Database\BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\code5\system\Database\BaseConnection.php(1485): CodeIgniter\Database\BaseConnection->query('SHOW TABLES FRO...')
#4 C:\xampp\htdocs\code5\system\Database\BaseConnection.php(1505): CodeIgniter\Database\BaseConnection->listTables()
#5 C:\xampp\htdocs\code5\system\Database\MigrationRunner.php(759): CodeIgniter\Database\BaseConnection->tableExists('migrations')
#6 C:\xampp\htdocs\code5\system\Database\MigrationRunner.php(163): CodeIgniter\Database\MigrationRunner->ensureTable()
#7 C:\xampp\htdocs\code5\system\Commands\Database\Migrate.php(85): CodeIgniter\Database\MigrationRunner->latest(NULL)
#8 C:\xampp\htdocs\code5\system\CLI\Commands.php(70): CodeIgniter\Commands\Database\Migrate->run(Array)
#9 C:\xampp\htdocs\code5\system\CLI\Console.php(48): CodeIgniter\CLI\Commands->run('migrate', Array)
#10 C:\xampp\htdocs\code5\system\Boot.php(351): CodeIgniter\CLI\Console->run()
#11 C:\xampp\htdocs\code5\system\Boot.php(104): CodeIgniter\Boot::runCommand(Object(CodeIgniter\CLI\Console))
#12 C:\xampp\htdocs\code5\spark(84): CodeIgniter\Boot::bootSpark(Object(Config\Paths))
#13 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Unknown database 'employee_form' in C:\xampp\htdocs\code5\system\Database\MySQLi\Connection.php:231
Stack trace:
#0 C:\xampp\htdocs\code5\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\code5\system\Database\BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\code5\system\Database\BaseConnection.php(1485): CodeIgniter\Database\BaseConnection->query('SHOW TABLES FRO...')
#3 C:\xampp\htdocs\code5\system\Database\BaseConnection.php(1505): CodeIgniter\Database\BaseConnection->listTables()
#4 C:\xampp\htdocs\code5\system\Database\MigrationRunner.php(759): CodeIgniter\Database\BaseConnection->tableExists('migrations')
#5 C:\xampp\htdocs\code5\system\Database\MigrationRunner.php(163): CodeIgniter\Database\MigrationRunner->ensureTable()
#6 C:\xampp\htdocs\code5\system\Commands\Database\Migrate.php(85): CodeIgniter\Database\MigrationRunner->latest(NULL)
#7 C:\xampp\htdocs\code5\system\CLI\Commands.php(70): CodeIgniter\Commands\Database\Migrate->run(Array)
#8 C:\xampp\htdocs\code5\system\CLI\Console.php(48): CodeIgniter\CLI\Commands->run('migrate', Array)
#9 C:\xampp\htdocs\code5\system\Boot.php(351): CodeIgniter\CLI\Console->run()
#10 C:\xampp\htdocs\code5\system\Boot.php(104): CodeIgniter\Boot::runCommand(Object(CodeIgniter\CLI\Console))
#11 C:\xampp\htdocs\code5\spark(84): CodeIgniter\Boot::bootSpark(Object(Config\Paths))
#12 {main}
ERROR - 2024-09-30 02:38:45 --> Error connecting to the database: mysqli_sql_exception: Unknown database 'employee_form' in C:\xampp\htdocs\code5\system\Database\MySQLi\Connection.php:186
Stack trace:
#0 C:\xampp\htdocs\code5\system\Database\MySQLi\Connection.php(186): mysqli->real_connect('localhost', 'root', Object(SensitiveParameterValue), 'employee_form', 3306, '', 0)
#1 C:\xampp\htdocs\code5\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\code5\system\Database\BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\code5\system\Database\BaseConnection.php(1485): CodeIgniter\Database\BaseConnection->query('SHOW TABLES FRO...')
#4 C:\xampp\htdocs\code5\system\Database\BaseConnection.php(1505): CodeIgniter\Database\BaseConnection->listTables()
#5 C:\xampp\htdocs\code5\system\Database\MigrationRunner.php(759): CodeIgniter\Database\BaseConnection->tableExists('migrations')
#6 C:\xampp\htdocs\code5\system\Database\MigrationRunner.php(163): CodeIgniter\Database\MigrationRunner->ensureTable()
#7 C:\xampp\htdocs\code5\system\Commands\Database\Migrate.php(85): CodeIgniter\Database\MigrationRunner->latest(NULL)
#8 C:\xampp\htdocs\code5\system\CLI\Commands.php(70): CodeIgniter\Commands\Database\Migrate->run(Array)
#9 C:\xampp\htdocs\code5\system\CLI\Console.php(48): CodeIgniter\CLI\Commands->run('migrate', Array)
#10 C:\xampp\htdocs\code5\system\Boot.php(351): CodeIgniter\CLI\Console->run()
#11 C:\xampp\htdocs\code5\system\Boot.php(104): CodeIgniter\Boot::runCommand(Object(CodeIgniter\CLI\Console))
#12 C:\xampp\htdocs\code5\spark(84): CodeIgniter\Boot::bootSpark(Object(Config\Paths))
#13 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Unknown database 'employee_form' in C:\xampp\htdocs\code5\system\Database\MySQLi\Connection.php:231
Stack trace:
#0 C:\xampp\htdocs\code5\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\code5\system\Database\BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\code5\system\Database\BaseConnection.php(1485): CodeIgniter\Database\BaseConnection->query('SHOW TABLES FRO...')
#3 C:\xampp\htdocs\code5\system\Database\BaseConnection.php(1505): CodeIgniter\Database\BaseConnection->listTables()
#4 C:\xampp\htdocs\code5\system\Database\MigrationRunner.php(759): CodeIgniter\Database\BaseConnection->tableExists('migrations')
#5 C:\xampp\htdocs\code5\system\Database\MigrationRunner.php(163): CodeIgniter\Database\MigrationRunner->ensureTable()
#6 C:\xampp\htdocs\code5\system\Commands\Database\Migrate.php(85): CodeIgniter\Database\MigrationRunner->latest(NULL)
#7 C:\xampp\htdocs\code5\system\CLI\Commands.php(70): CodeIgniter\Commands\Database\Migrate->run(Array)
#8 C:\xampp\htdocs\code5\system\CLI\Console.php(48): CodeIgniter\CLI\Commands->run('migrate', Array)
#9 C:\xampp\htdocs\code5\system\Boot.php(351): CodeIgniter\CLI\Console->run()
#10 C:\xampp\htdocs\code5\system\Boot.php(104): CodeIgniter\Boot::runCommand(Object(CodeIgniter\CLI\Console))
#11 C:\xampp\htdocs\code5\spark(84): CodeIgniter\Boot::bootSpark(Object(Config\Paths))
#12 {main}
ERROR - 2024-09-30 05:38:42 --> mysqli_sql_exception: Can't create database 'employee_info'; database exists in C:\xampp\htdocs\employee\system\Database\MySQLi\Connection.php:312
Stack trace:
#0 C:\xampp\htdocs\employee\system\Database\MySQLi\Connection.php(312): mysqli->query('CREATE DATABASE...', 0)
#1 C:\xampp\htdocs\employee\system\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('CREATE DATABASE...')
#2 C:\xampp\htdocs\employee\system\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('CREATE DATABASE...')
#3 C:\xampp\htdocs\employee\system\Database\Forge.php(232): CodeIgniter\Database\BaseConnection->query('CREATE DATABASE...')
#4 C:\xampp\htdocs\employee\system\Commands\Database\CreateDatabase.php(136): CodeIgniter\Database\Forge->createDatabase('employee_info')
#5 C:\xampp\htdocs\employee\system\CLI\Commands.php(70): CodeIgniter\Commands\Database\CreateDatabase->run(Array)
#6 C:\xampp\htdocs\employee\system\CLI\Console.php(48): CodeIgniter\CLI\Commands->run('db:create', Array)
#7 C:\xampp\htdocs\employee\system\Boot.php(351): CodeIgniter\CLI\Console->run()
#8 C:\xampp\htdocs\employee\system\Boot.php(104): CodeIgniter\Boot::runCommand(Object(CodeIgniter\CLI\Console))
#9 C:\xampp\htdocs\employee\spark(84): CodeIgniter\Boot::bootSpark(Object(Config\Paths))
#10 {main}
